src/test/java/com/example/demo/service/mapper/UserMapperTest.java

```diff
@@ -0,0 +1,44 @@
+package com.example.demo.service.mapper;
+
+import com.example.demo.model.ReviewDto;
+import com.example.demo.model.User;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import org.junit.Assert;
+import static org.junit.Assert.*;
```

Comment:
avoid star imports and place static imports before non-static, with import blocks separated with a single empty line

Reply:
done